struct.go

```diff
@@ -6,13 +6,13 @@ import (
 	"time"
 )
 
-// EurekaUrlSlugs is a map of resource names -> eureka URLs
+// EurekaUrlSlugs is a map of resource names→Eureka URLs.
```

Comment:
Done. Note that I did use some em dashes (—) as well, which are in Latin-1 but not ASCII. Do you want me to avoid those as well?